tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py{27,35}-django{18,19}, quality
```

Comment:
We are running all combinations: https://travis-ci.org/edx/edx-drf-extensions/builds/116194344.